tests/test_scorecard.py

```diff
@@ -0,0 +1,14 @@
+# tests/test_scorecard.py
+import unittest
+from src.ossf_scorecard.scorecard import get_scorecard
+
+class TestScorecard(unittest.TestCase):
+    def test_get_scorecard(self):
+        platform = "github.com"
+        org = "nexB"
+        repo = "scancode-toolkit"
+        data = get_scorecard(platform, org, repo)
+        self.assertIn("scorecard", data)
+
+if __name__ == "__main__":
```

Comment:
We do not run test file directly, we should us pytest.
These tests should also be added and run on the CI here so we can see it passes.

Reply:
this is also resolved